.sqlfluff

```diff
@@ -0,0 +1,22 @@
+[sqlfluff]
+dialect = postgres
+templater = placeholder
+exclude_rules =
+  layout.indent,
```

Comment:
Indentation is more complicated, more controversial, and requires a bigger change than the others in this pull request, so I'm holding it for another PR.